Content.Server/Tabletop/TabletopSystem.Chess.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Content.Shared.DrawDepth;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+
+namespace Content.Server.Tabletop
+{
+    public partial class TabletopSystem
+    {
+        private void SetupChessBoard(MapId mapId)
+        {
+            var chessboard = _entityManager.SpawnEntity("ChessBoardTabletop", new MapCoordinates(-1, 0, mapId));
+
+            if (chessboard.TryGetComponent<SpriteComponent>(out var spriteComponent))
+            {
+                spriteComponent.Scale = new Vector2(1, 1);
+                spriteComponent.DrawDepth = (int) DrawDepth.FloorTiles;
+            }
+
+            chessboard.Transform.Anchored = true;
```

Comment:
why cant these just be on the prototype?

Reply:
Moved DrawDepth to prototype, I think I left in scale accidentally because I was scaling my 16x16 px codersprite before.

I left anchored like this for now because I don't see a better way of doing it.